### PR TITLE
Text changes to "publish option" field in "create" form

### LINKED ISSUE #440: Text changes to "publish option" field in "create" form
Currently, the last option in the "create" project form has a confusing language. 

Screen Shot 2022-04-21 at 5 44 54 PM

How about we change "What publish option do you want to set for this project? " to "Ready to publish your project?"

And change the help text from  "Think about your target audience. Should this project be visible to all creators?, authenticated creators?, or do you want to provide the usernames of creators this project should be visible to?" to "Choose an option from the dropdown below".

Also change "Authenticated Creators" to "ZubHub Users"

### LINKED ISSUE #441: UX enhancements to "publish option" in "create" form
First, some text changes are needed in the "publish option" field, issue is filed here: https://github.com/unstructuredstudio/zubhub/issues/440. 

Second comes UX improvements, which have primarily to do with options in the dropdown. I think the four options are confusing and not entirely related to publishing. 

I think the dropdown should only have two options, "Public" and "ZubHub Users". "Draft" can be moved out of this dropdown, as saving the project in draft mode means the changes are still in progress and not published. So either "Draft" should be named "Private" or appear next to the "Create Project" button as "Save as draft" and "Create Project" can be changed to "Publish Project" or "Publish Changes".

Then, the "Preview" option in its current shape and form is very confusing. It doesn't convey its intended purpose, which is that it allows anyone to share their project with other users for preview. One idea - this option can be offered as soon as the project is saved below the create form itself, just like how blogging platforms like Ghost allow.

Screen Shot 2022-04-21 at 5 44 54 PM

### PR DESCRIPTION
**Pull request by godsakani**
Closes #440, Closes #441

## Changes

- Changed "What publish option do you want to set for this project? " to "Ready to publish your project?"
- Changed "Think about your target audience. Should this project be visible to all creators?, authenticated creators?, or do you want to provide the usernames of creators this project should be visible to?" to "Choose an option from the dropdown below".
- Changed "Authenticated Creators" to "ZubHub Users"

## Screenshots
fixed

### CODE REVIEW

**Comment by godsakani** [comment]
Hello @srish , @tuxology , @NdibeRaymond , @ajain1921 
Please can check the PR

**Comment by tuxology** [comment]
Thanks @godsakani I was thinking we should make the question more easy to understand. Here is what I think UX should be   like:

(8) Who all can see your project after you create it?
If you are still working on the project and are unsure if you want to publish this right away, select "Only me". 

- Everyone
- All ZubHub Users
- Only a few ZubHub Users
- Only me for now

Let me explain what all these are equivalent to currently:
**Everyone** is same as **Public**
**All ZubHub Users** is same as **Authenticated Users**
**Only a few Zubhub Users** is same as **Private**
**Only me for now** is same as **Draft**

In upcoming changes, **Draft** mode will be dropped from this list and will have a separate button so we will just have 3 options later.

File: zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js [hunk 1]
```diff
@@ -105,16 +105,29 @@ const styles = theme => ({
       height: '3.5em',
       width: '100%',
       flexDirection: 'row',
-      justifyContent: 'flex-start',
+      justifyContent: 'left',
```
Comment (tuxology) [inline zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js:108]: Why do we need changes here? Can you attach a quick screenshot?

File: zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js [hunk 2]
```diff
@@ -105,16 +105,29 @@ const styles = theme => ({
...
     },
   },
   actionBoxButtonStyle: {
```
Comment (tuxology) [inline zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js:111]: Why do we need actionBoxButton changes as part of this PR? Can you attach a screenshot?

**Review by tuxology — COMMENTED** [review]
Left another comment about text changes. Can we do those as well.

File: zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js (hunk 1 above)
Comment (godsakani) [reply zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js:108]: ### 

> Why do we need changes here? Can you attach a quick screenshot?

sorry about that sir I didn't notice. this change was done on a previous issues i worked on which we had to close the PR .

**Comment by godsakani** [comment]
Hello @tuxology , @srish , @ajain1921 , @NdibeRaymond 

I am closing this issue to reopen it back due to the unwanted files added to it during commit

**Pull request closed without merging**